conan.cmake

```diff
@@ -142,6 +142,12 @@ function(conan_cmake_settings result)
         conan_cmake_detect_vs_runtime(_vs_runtime)
         message(STATUS "Detected VS runtime: ${_vs_runtime}")
         set(_SETTINGS ${_SETTINGS} -s compiler.runtime=${_vs_runtime})
+        
+        if (CMAKE_GENERATOR_TOOLSET)
+            set(_SETTINGS ${_SETTINGS} -s compiler.toolset=${CMAKE_GENERATOR_TOOLSET})
+        elseif(CMAKE_VS_PLATHORM_TOOLSET AND (CMAKE_GENERATOR STREQUAL "Ninja"))
```

Comment:
This looks like a typo, should be `CMAKE_VS_PLATFORM_TOOLSET` (F in platform, not H)

Reply:
Thank, fix